mllib-local/pom.xml

```diff
@@ -66,6 +66,11 @@
       <groupId>org.apache.spark</groupId>
       <artifactId>spark-tags_${scala.binary.version}</artifactId>
     </dependency>
+    <dependency>
```

Comment:
I am still a bit worried about the dependency change.
Defer to @WeichenXu123 's review.

Reply:
the dep change can be eliminated if logging is unnecessary.